oat-sa/extension-tao-itemqti
# [Fix] TAO 9843 - Media interaction playback continues after leaving page

**Related to task:**  https://oat-sa.atlassian.net/browse/TAO-9843
**Description:** Media interaction: playback continues after leaving page if autostart is set
**How to test:** 
- Create an item with Media interaction
- Upload audio/video file
- Set 'Autostart' in the Interaction Properties
- Save and Preview

**Expected result:** playback autostarted in authoring mode finishes as soon as Preview mode is opened.
